dashboard/client/components/app.tsx

```diff
@@ -46,7 +46,7 @@ class App extends React.Component {
   };
 
   render() {
-    const homeUrl = configStore.isClusterAdmin ? clusterURL() : workloadsURL();
+    const homeUrl = clusterURL();
```

Comment:
If we show "Cluster" to all users then we probably need to add "allowedResources" checks to overview page?

Reply:
The only two items on the overview page appeared to be metrics, which already gracefully degrade, and cluster-level warning events. What sort of check would you envision there?

Reply:
I think we should check if user has access to nodes (overview warnings are also looking node conditions).